thin_client/thin_client_main.py

```diff
@@ -0,0 +1,111 @@
+import pygame
+import sys
+from pygame.locals import * 
+import thin_client.vlc_addon as vlc_addon
+import thin_client.networking as networking
+import thin_client.settings as s
+    
+def initializePygame(FPS):
+    pygame.init()
+    screen = pygame.display.set_mode((s.RESO_WIDTH, s.RESO_HEIGHT))
+    pygame.display.set_caption("Remote Keyboard")
+    pygame.mouse.set_visible(False) # Makes mouse invisible
```

Comment:
since you have `settings`, might as well put all those constant into `settings` global variable. such as:

* `FPS`
* `DISPLAY_CAPTION`
* etc...